New content for control using AMDC

npetersen2 opened the pull request:
So far, this PR adds the structure given in #77 

<img src="https://github.com/Severson-Group/docs.amdc.dev/assets/20168990/73517b90-3eee-4191-9618-3afc8ad8b0fb" width="350px" />

<img src="https://github.com/Severson-Group/docs.amdc.dev/assets/20168990/9e63b25e-81a0-4495-bcd5-1b82eeecc64d" width="300px" />

npetersen2 committed:
Add initial template of new docs control content per #77

npetersen2 commented:
- @anirudhupadhyaya  -- encoder feedback
- @aravind075 -- PWM dead-time compensation
- @noguchi-takahiro -- anti-windup

noguchi-takahiro commented:
I added new PR https://github.com/Severson-Group/docs.amdc.dev/pull/90. Please review it for your convenience.

anirudhupadhyaya committed:
initial commit

elsevers committed:
Update index.md

npetersen2 commented:
Here is a report on PWM dead-time compensation: 
[00 - Dead Time Compensation.pdf](https://github.com/Severson-Group/docs.amdc.dev/files/14746961/00.-.Dead.Time.Compensation.pdf)

anirudhupadhyaya committed:
Add content

anirudhupadhyaya committed:
Update docs

anirudhupadhyaya committed:
Update doc

anirudhupadhyaya committed:
Update docs

anirudhupadhyaya committed:
Update docs

anirudhupadhyaya committed:
Update doc

anirudhupadhyaya committed:
Update block diagram

anirudhupadhyaya committed:
Apply suggestions from code review

Co-authored-by: Eric Severson <eric.severson@wisc.edu>

anirudhupadhyaya committed:
Update doc to address review comments

anirudhupadhyaya committed:
Add line on low pass filter

anirudhupadhyaya committed:
Update source/getting-started/control-with-amdc/encoder-fb/index.md

Co-authored-by: Eric Severson <eric.severson@wisc.edu>

anirudhupadhyaya committed:
Add current sensor calibration documentation (#95)

* Initial commit

* Update readme

* Update readme

* Update readme

* Add files

* Update files

* Changes

* Changes

* Update files

* Get rif of scope image

* Minor updates

* Address review comments

* Remove resources

* Add back resources

* Address review comments

* Address review comments

* Update index.md

* Edit current sensor calibration method section

* Edit calibration method section

* Add section on Recalcilating current sensor offset

* Clarify that we measure a voltage.

* Edit conclusion

* Update index.md

* Update index.md

* Address review comments

* Update index.md

* Update index.md

* Update index.md

* Apply suggestions from code review

Co-authored-by: Eric Severson <eric.severson@wisc.edu>

* Update source/getting-started/control-with-amdc/current-sensor-cal/index.md

* Update source/getting-started/control-with-amdc/current-sensor-cal/index.md

---------

Co-authored-by: Nathan Petersen <npetersen2@wisc.edu>
Co-authored-by: Eric Severson <eric.severson@wisc.edu>

elsevers committed:
Edit background

elsevers committed:
Fix file name type-o

elsevers committed:
Fix type-os

elsevers committed:
Add config instructions

elsevers committed:
Edit first 1/3 of document

elsevers committed:
Fix current calibration images (#116)

elsevers committed:
add simulink model of pll

noguchi-takahiro committed:
Add pll test files

Daehoon-Sung committed:
Change indexme file

Daehoon-Sung committed:
Update index.md

elsevers committed:
Edit finding offset and step 1

elsevers committed:
Edit finding the offset section

elsevers committed:
Add encoder angle diagram, first edits of text to use it.

elsevers committed:
Add latex source for motor cross section

elsevers committed:
add test code

elsevers committed:
Merge branch 'main' into add-control-content

# Conflicts:
#	source/getting-started/control-with-amdc/current-sensor-cal/index.md
#	source/getting-started/control-with-amdc/index.md
#	source/index.rst

elsevers committed:
Merge branch 'add-control-content' into add-enc-anirudh

elsevers committed:
- Edit everything before finding the offset

- Fix links

elsevers committed:
Merge branch 'add-enc-anirudh' into user/Daehoon-Sung/update-offset-report

elsevers committed:
change image naming to dash-case

elsevers committed:
Improve typesetting around the image

elsevers committed:
Add torque characteristic and update step 1 instructions.

elsevers committed:
Make motor cross-section plot 2 poles, add phave v and w axes

elsevers committed:
Merge branch 'add-enc-anirudh' into user/Daehoon-Sung/update-offset-report

elsevers committed:
Add clarifying comment to torque characteristic

Daehoon-Sung committed:
Update index.md

Apply feedback from Professor's comment (option 2) to revise the steps to find offset

Daehoon-Sung committed:
Update the encoder offset plot

Daehoon-Sung committed:
Update index.md

Update the image

Daehoon-Sung committed:
Update index.md

Add the formula

Daehoon-Sung committed:
Update source/getting-started/control-with-amdc/encoder-fb/index.md

Co-authored-by: Takahiro <114006024+noguchi-takahiro@users.noreply.github.com>

Daehoon-Sung committed:
Update index.md

Daehoon-Sung committed:
Update index.md

Daehoon-Sung committed:
Update index.md

noguchi-takahiro committed:
Fix rendering of equation

noguchi-takahiro committed:
Fix rendering issue of image

Daehoon-Sung committed:
Update the encoder image

Daehoon-Sung committed:
Merge branch 'user/Daehoon-Sung/update-offset-report' of https://github.com/Severson-Group/docs.amdc.dev into user/Daehoon-Sung/update-offset-report

Daehoon-Sung committed:
Update index.md

Update the variable names

Daehoon-Sung committed:
Update the figure

Daehoon-Sung committed:
Clarify terminology for mechanical angle in documentation

Daehoon-Sung committed:
Enhance encoder offset determination section

Added detailed explanation and equations for determining encoder offset using closed-loop control and voltage measurements.

Daehoon-Sung committed:
Clarify voltage equation with angular velocity note

Added clarification about electrical angular velocity in the voltage equation.

Daehoon-Sung committed:
Clarify voltage vector description in index.md

Daehoon-Sung committed:
Fix voltage vector equation formatting

Corrected the formatting of the voltage vector equation in complex vector form.

Daehoon-Sung committed:
Refine description of encoder offset estimation process

Clarified the conditions for estimating encoder offset by refining language regarding the alignment of the gamma-delta and d-q frames.

Daehoon-Sung committed:
Fix minor grammatical error in encoder offset section

Daehoon-Sung committed:
Fix formatting in encoder offset determination section

Daehoon-Sung committed:
Revise current command notation in encoder equations

Updated notation for current commands in the equation for v_gamma.

noguchi-takahiro committed:
Fix rendering issues for images

Daehoon-Sung committed:
Clarify angle calculation and encoder offset procedure

Updated explanations regarding the calculation of the motor's angle and the determination of the encoder offset. Improved clarity on the relationship between electrical angle and voltage vector.

elsevers committed:
Revise encoder offset section contents (#146)

* Edit encoder offset precise section

* Improve typesetting and clarify vd

elsevers committed:
Merge pull request #139 from Severson-Group/user/Daehoon-Sung/update-offset-report

Update offset section in the encoder article

elsevers committed:
fix typo

elsevers committed:
clean up reference

elsevers committed:
cleanup msc lint errors

elsevers committed:
fix reference warning and eq typsetting

elsevers committed:
rename files to dash case

elsevers committed:
fix typo

elsevers committed:
replace encoder block diagram

elsevers committed:
Merge pull request #103 from Severson-Group/add-enc-anirudh

Add encoder calibration documentation
